core/server/api/v2/utils/index.js

```diff
@@ -9,5 +11,9 @@ module.exports = {
 
     get validators() {
         return require('./validators');
+    },
+
+    isContentAPI: (frame) => {
```

Comment:
This refactoring happened before I dug out that this context check is not working any more passed `permissions` stage. Should probably be put in separate PR to be absolutely clean